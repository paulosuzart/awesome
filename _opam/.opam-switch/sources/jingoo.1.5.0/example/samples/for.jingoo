{%- for item in [ 0, 1, 2, 3, 4, 5 ] -%}
  - {{ item }}: {{ loop.cycle (["even","odd"]) }}
{% endfor -%}
{%- for item in links -%}
<a href="{{ item[0] }}">{{ item[1] }}</a>
{% endfor -%}
{%- for (href, title) in links2 -%}
<a href="{{ href }}">{{ title }}</a>
{% endfor -%}
{%- for (key, value) in obj -%}
  - {{ key }}: {{ value }}
{% endfor -%}
{% for s in str1 -%}
  {{ s }}({{ loop.index }})
{% endfor -%}
{% for s in str2 -%}
{{ s }}({{ loop.index }})
{% endfor -%}
