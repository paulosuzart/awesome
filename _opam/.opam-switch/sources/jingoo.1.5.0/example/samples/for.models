{{ { links: [ [ "http://yahoo.co.jp", "yahoo japan" ],
	      [ "http://google.co.jp", "google japan" ] ],
     links2:[ ("http://yahoo.co.jp", "yahoo japan"),
              ("http://google.co.jp", "google japan") ],
     obj: { name: "alice", age: 42, sex: "F" },
     str1: "abc",
     str2: "日本語"
   } }}
