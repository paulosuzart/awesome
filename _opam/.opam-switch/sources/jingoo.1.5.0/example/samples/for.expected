- 0: even
- 1: odd
- 2: even
- 3: odd
- 4: even
- 5: odd
<a href="http://yahoo.co.jp">yahoo japan</a>
<a href="http://google.co.jp">google japan</a>
<a href="http://yahoo.co.jp">yahoo japan</a>
<a href="http://google.co.jp">google japan</a>
- name: alice
- age: 42
- sex: F
a(1)
b(2)
c(3)
日(1)
本(2)
語(3)
